src/Domain/BlockLogicalCoordinates.cpp

```diff
@@ -153,22 +153,26 @@ BlockLogicalCoords<Dim> block_logical_coordinates_in_excision(
       continue;
     }
     // Discard block if the point has angular logical coordinates outside the
-    // range [-1, 1]
+    // range [-1, 1]. Also discard if the point is on the upper angular face of
+    // the block, to disambiguate which of the two blocks sharing the face the
+    // point is in.
```

Comment:
This doesn't work.  The blocks don't have aligned frames, so some interfaces are two lower sides or two upper sides.